redact/types.go

```diff
@@ -0,0 +1,40 @@
+package redact
+
+import (
+	"encoding/json"
+)
+
+type RedactedString struct {
+	inputString string
+	redactions  []*Redact
+
+	redactedString string
```

Comment:
Good call by the linter, but I had thought about using this to store the redacted string and then, if it's not empty, just displaying it out of the string method. I thought this could potentially save re-running redactors unnecessarily.